packages/hash/frontend/src/blocks/page/createPlaceholderPlugin/Placeholder.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, styled, Typography } from "@mui/material";
+
+const Command = styled(Box)(({ theme }) => ({
+  color: theme.palette.gray[50],
+  borderRadius: 3,
+  background: theme.palette.gray[10],
+  width: "1.875em",
+  height: "1.875em",
```

Comment:
Do these need to be specified? Isn't it clearer to specify the desired padding and have the size be dependent on `fontSize` and the content?

Reply:
Hmm, actually it's desired to be a square as far as I understood from the design. So maybe combination of vertical padding (`py`) and `aspect-ratio` could work better?

Reply:
Checked it again, looks like width-height works the best on this case, since we want to render a square with centered content, and square's size is relative to text-size. We can either use `width`/`height` or `width`/`aspect-ratio`. If there is another concern that I'm not aware of, let me know, otherwise I prefer keeping it like this.

Reply:
It doesn't really matter, but generally putting fixed width/height on boxes which could in theory have variable content is not advisable because you can end up with overflow. I just tend to keep CSS more flexible. I don't think it actually matters here though so will resolve.

Reply:
I got the point, thanks for the heads-up. And yes, this component is designed to always have one key (at least for now).